src/styles/cardBlocks/card.scss

```diff
@@ -0,0 +1,86 @@
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-weight: 400;
+  margin: 0;
+}
+
+p,
+h2 {
+  margin: 0;
+}
```

Comment:
not fixed:
![image](https://github.com/mate-academy/layout_product-cards/assets/86923089/1592fc01-306d-40e5-8916-74e56a04fca2)

it doesn't relate to the card block
move it into a separate file